src/Octoshift/Commands/GrantMigratorRole/GrantMigratorRoleCommandBase.cs

```diff
@@ -43,7 +47,8 @@ public override GrantMigratorRoleCommandHandler BuildHandler(GrantMigratorRoleCo
 
         var log = sp.GetRequiredService<OctoLogger>();
         var githubApiFactory = sp.GetRequiredService<ITargetGithubApiFactory>();
-        var githubApi = githubApiFactory.Create(args.GhesApiUrl, args.GithubPat);
+        var api_url = args.TargetApiUrl ?? args.GhesApiUrl;
```

Comment:
I didn't know if we should keep `GhesApiUrl` I guess the naming is there for GHES source where we need to grant migrator role there, sowe can keep it for now and can add an override note 👍🏾 .

Reply:
Discussed with @ArinGhazarian on call and will preserve both flags (and validate one or the other) for use cases as outlined below and backwards compatibility and make clearer descriptions. Eventually we might want to deprecate into one flag but this can be a future enhancement.
```
// GHEC -> GHEC (target-api-url) grant migrator role for import and export
// GHES -> GHEC (target-api-url) grant migrator role for import (ghes-api-url) grant migrator role for export
// GHEC -> PROXIMA (target-api-url) grant migrator role for import, run without option to grant migrator role for export
// GHES -> PROXIMA (target-api-url) grant migrator role for import, (ghes-api-url) grant migrator role for export
// Proxima -> GHEC run without option to  grant migrator role for import, (target-api-url) grant migrator role for export
```